lib/bob/job/build_otp.ex

```diff
@@ -1,10 +1,10 @@
 defmodule Bob.Job.BuildOTP do
   require Logger
 
-  def run(ref_name, ref, linux) do
+  def run(ref_name, ref, linux, arch) do
```

Comment:
`Bob.Queue.add({Bob.Job.BuildOTP, arch}, [ref_name, ref, linux])` will add `arch` as the first argument to the job.

```suggestion
  def run(arch, ref_name, ref, linux) do
```